index.html

```diff
@@ -1477,9 +1477,11 @@
       </h2>
       <p>
         The editors would like to express their gratitude to the Mozilla and Telefónica Digital
-        teams implementing the Firefox OS Push message solution and specially to Doug Turner,
-        Nikhil Marathe, Fernando R. Sela, Guillermo López, Antonio Amaya, José Manuel Cantera and
-        Albert Crespell, for their technical guidance, implementation work and support.
+        teams implementing the Firefox OS Push message solution, as well as to the following people
+        who provided significant technical input to this document: Antonio Amaya, Kit Cambridge,
```

Comment:
please add Ben Bangert and Phil Jenvey (both are part of the mozilla team and have provided guidance and suggestions)